### PR TITLE
LPS-103632 BuildLang

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:stable

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/asset/asset-auto-tagger-service/src/test/java/com/liferay/asset/auto/tagger/internal/configuration/persistence/listener/AssetAutoTaggerCompanyConfigurationModelListenerTest.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+/**
+ * @author Alicia Garcia
+ */
+@PrepareForTest(ResourceBundleUtil.class)
+@RunWith(PowerMockRunner.class)
+public class AssetAutoTaggerCompanyConfigurationModelListenerTest {
+
+	@Before
+	public void setUp() {
+		MockitoAnnotations.initMocks(this);
```
Comment (adolfopa) [inline modules/apps/asset/asset-auto-tagger-service/src/test/java/com/liferay/asset/auto/tagger/internal/configuration/persistence/listener/AssetAutoTaggerCompanyConfigurationModelListenerTest.java]: This needs a line feed.

File: modules/apps/asset/asset-auto-tagger-service/src/test/java/com/liferay/asset/auto/tagger/internal/configuration/persistence/listener/AssetAutoTaggerCompanyConfigurationModelListenerTest.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+		_setUpResourceBundleUtil();
+	}
+
+	@Test(expected = ConfigurationModelListenerException.class)
+	public void testMaximumNumberOfTagsPerAssetGreaterThanSystem()
+		throws ConfigurationModelListenerException {
+
+		_setUpAssetAutoTaggerConfigurationFactory();
+
+		ReflectionTestUtil.setFieldValue(
```
Comment (adolfopa) [inline modules/apps/asset/asset-auto-tagger-service/src/test/java/com/liferay/asset/auto/tagger/internal/configuration/persistence/listener/AssetAutoTaggerCompanyConfigurationModelListenerTest.java]: We need to make sure we undo any side effects to make sure any other tests are not affected.

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
Just started reviewing :)

**Comment by adolfopa** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/80654.

**Pull request closed without merging**